File: BancoHistorico/BancoHistorico/Fk.cs
using System;
using System.ComponentModel.DataAnnotations;
using System.ComponentModel.DataAnnotations.Schema;

namespace BancoHistorico.Cadastros.Domain
{
    public class Fk
    {
        [Key]
        public Guid Id { get; set; }

        [Column("referenced_object_id")]
        public int referenced_object_id { get; private set; }
        [Column("referenced_column_id")]
        public int referenced_column_id { get; private set; }
        [Column("FK_CONSTRAINT")]
        public string FK_CONSTRAINT { get; private set; }
        [Column("referenced_column")]
        public string referenced_column { get; private set; }
        [Column("referenced_object")]
        public string referenced_object { get; private set; }
        [Column("parent_object")]
        public string parent_object { get; private set; }
        [Column("parent_column")]
        public string parent_column { get; private set; }
    }
}

File: EntityFrm/Danps/scripts/darts/fk.dart

import 'package:json_annotation/json_annotation.dart';

part 'fk.g.dart';

@JsonSerializable()
class Fk {
	int referenced_object_id;
	int referenced_column_id;
	String fK_CONSTRAINT;
	String referenced_column;
	String referenced_object;
	String parent_object;
	String parent_column;

	Fk({required this.referenced_object_id,required this.referenced_column_id,required this.fK_CONSTRAINT,required this.referenced_column,required this.referenced_object,required this.parent_object,required this.parent_column,});

	factory Fk.fromJson(Map<String, dynamic> json) => _$FkFromJson(json);

	Map<String, dynamic> toJson() => _$FkToJson(this);
}